Regression: Typescript error in antd 3.16.4

- [ x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://typescriptlang.com](https://typescriptlang.com)

### Steps to reproduce
Typescript 3.4.4

importing and using 'antd' components in your projects results in following compilation error

### What is expected?
no error

### What is actually happening?

```
node_modules/antd/lib/form/interface.d.ts:2:8 - error TS1192: Module '"/Users/pdeva/code/dashboard/node_modules/@types/hoist-non-react-statics/index"' has no default export.

2 import hoistNonReactStatics from 'hoist-non-react-statics';
```

| Environment | Info |
|---|---|
| antd | 3.16.4 |
| React | 16.8.6 |
| System | os x mavericks |
| Browser | chrome |


<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/components/form/interface.ts
+++ b/components/form/interface.ts
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import hoistNonReactStatics from 'hoist-non-react-statics';
+import * as hoistNonReactStatics from 'hoist-non-react-statics';
 import { Omit } from '../_util/type';
 import { FormComponentProps } from './Form';
 

